ui/app/src/components/DashletCard/DashletCard.tsx

```diff
@@ -40,10 +48,13 @@ export type DashletCardProps = PropsWithChildren<
       footer: BoxProps['sx'];
     }>;
     cardContentProps?: CardContentProps;
+    maximizable?: boolean;
+    collapsible?: boolean;
```

Comment:
Collapsible is needed for it not to be shown in the activityDashlet (in desktopMode). Defaults to true